modules/mint-mode-with-removal-of-admin-credential.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/manually-creating-iam.adoc
+
+[id="mint-mode-with-removal-or-rotation-of-admin-credential_{context}"]
```

Comment:
Is this doc section AWS specific? The feature is only available on AWS right now, though we hope to land it for Azure and GCP soon.

Reply:
Yes, I am setting this up so that it only appears in the AWS section.